scripts/zkai_native_seq32_attention_mlp_adjacent_label_seed_sweep_gate.py

```
@@ -0,0 +1,680 @@
+        "proof_sha256": accounting_row["proof_sha256"],
+        "record_stream_sha256": accounting["record_stream_sha256"],
+    }
+
+
+def build_rows() -> list[dict[str, Any]]:
+    accounting = accounting_rows_by_path()
+    rows = []
+    for expected in EXPECTED_ROWS:
+        row = accounting.get(expected["path"])
+        if row is None:
+            raise AdjacentSeedSweepGateError(f"missing accounting row: {expected['path']}")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)